model/installation_states.go

```diff
@@ -221,6 +221,9 @@ var (
 			InstallationStateDeletionPendingRequested,
 			InstallationStateDeletionPendingInProgress,
 		},
+		InstallationStateDeletionPending: {
+			InstallationStateDeletionPending,
```

Comment:
Is this the only state this can move to?

Reply:
Yep! That's right.